design/ceph/ceph-csi-driver.md

```diff
@@ -142,3 +142,9 @@ Currently the code requires changing numerous parameters to enable CSI.  This
 document aims to change that to a single parameter. In the future it may be
 desirable to make this more of a "runtime" parameter that could be managed in
 the cluster CRD.
+
+### Support RBD Mirroring
+
+To support RBD Mirroring, the [Volume Replication Operator](https://github.com/csi-addons/volume-replication-operator/blob/main/README.md) will be started in the RBD provisioner pod.
+Volume Replication Operator is a kubernetes operator that provides common and reusable APIs for storage disaster recovery. It is based on [csi-addons/spec](https://github.com/csi-addons/spec) specification and can be used by any storage provider.
+It follows controller pattern and provides extended APIs for storage disaster recovery. The extended APIs are provided via Custom Resource Definition (CRD).
```

Comment:
Let's document how they would enable the replication controller, perhaps just to link to the helm doc and mention the env var in the operator.yaml if they aren't using helm